### PR TITLE
docker-fedora: use /etc/docker/daemon.json as docker-centos

### PR DESCRIPTION
**Pull request by giuseppe**
sync with docker-centos the change in configuration.  Marked WIP as I have not yet tested it

### CODE REVIEW

**Comment by giuseppe** [comment]
removed the WIP from the title as I've tested it on Fedora.

File: docker-fedora/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 
-LABEL maintainer="Giuseppe Scrivano <gscrivan@redhat.com>"
+LABEL maintainer="Giuseppe Scrivano <gscrivan@redhat.com>" \
+      name="docker-fedora" \
+      version="0.1" \
+      atomic.type="system" \
+      architecture="x86_64"
 
-RUN dnf install -y docker container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin lvm2 iptables procps-ng xz oci-register-machine && mkdir -p /usr/lib/modules && dnf clean all
+RUN dnf install --setopt=tsflags=nodocs -y docker container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin lvm2 iptables procps-ng xz oci-register-machine && rpm -V docker container-selinux cloud-utils-growpart python-docker-py docker-novolume-plugin lvm2 iptables procps-ng xz oci-register-machine && mkdir -p /usr/lib/modules && dnf clean all
```
Comment (ashcrow) [inline docker-fedora/Dockerfile]: nit: Putting these on separate lines make it much more readable.

**Comment by ashcrow** [comment]
👍

**Comment by ashcrow** [comment]
@rh-atomic-bot r+ af33ff9

**Pull request closed without merging**